### PR TITLE
Dev 1019

### PR DESCRIPTION
**Pull request by mohamedveron**

### CODE REVIEW

File: executor/executor.go [hunk 1]
```diff
@@ -328,9 +330,43 @@ func (executor *Executor) execute(
...
+		statusMap := make(map[string]string)
+		statusMap["Running"] = "pod restarted successfully"
+		statusMap["Failed"] = "pod failed to restart"
+		statusMap["Unknown"] = "pod status is unknown"
+		statusMap["PodInitializing"] = "pod restarting"
+
+		backoff := 15 * time.Minute
+		msg := "pod restarting exceeded timout (15 min)"
+		start := time.Now()
+		timeout := int(backoff.Seconds())
```
Comment (iahvector) [inline executor/executor.go:345]: No need to do that. `time.Now() - start < backoff` should work

Comment (iahvector) [reply executor/executor.go:345]: Other than that it's fine, when you change that you can merge without waiting for me to review it again

Comment (mohamedveron) [reply executor/executor.go:345]: because the timeout variable not from type time

Comment (iahvector) [reply executor/executor.go:345]: You don't need the `timeout` variable at all. The `backoff` variable should be enough, like I used it in my example

Comment (mohamedveron) [reply executor/executor.go:345]: you are right but time.Now() is not a comparable unit it is of type time so you have to choose a unit to use this operation (-) and the most suitable one is seconds as we make short pooling every 15 second

**Pull request merged**